bootstrap/README.md

```diff
@@ -7,7 +7,7 @@ Perform the following operations to prepare your Raspberry Pi for becoming a TJB
 
 2. Run the command below:
 
-    $ curl -sL https://raw.githubusercontent.com/ibmtjbot/tjbot/master/bootstrap/bootstrap.sh | sudo -E bash -
+    curl -sL https://raw.githubusercontent.com/ibmtjbot/tjbot/master/bootstrap/bootstrap.sh | sudo -E bash -
```

Comment:
Nice to see the $ removed to make cut and paste easier :+1: 

I'm sorry I didn't spot this the last time, but if I'm not mistaken `sudo` shouldn't be used here as it will result in unwanted behavior when `npm install` is called from `bootstrap.sh`. Remember, `npm install` should be called without `sudo`.